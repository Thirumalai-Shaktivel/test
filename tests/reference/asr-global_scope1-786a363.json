{
    "basename": "asr-global_scope1-786a363",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope1.f90",
    "infile_hash": "449b38487b6c79754deedb9991f1867bcf8c67d0d93019c2fe15854c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope1-786a363.stdout",
    "stdout_hash": "8103c03b63b30900466141dc63715d62344bea11c55952247981434a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}