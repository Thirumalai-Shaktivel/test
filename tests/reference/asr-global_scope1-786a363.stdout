(TranslationUnit (SymbolTable 6cabfe67 {x: (Variable 6cabfe67 x Local (Integer 4 []))}) [])
